js/kucoin.js

```diff
@@ -3093,6 +3107,7 @@ module.exports = class kucoin extends Exchange {
          * @param {object} params extra parameters specific to the kucoin api endpoints
          * @param {string} params.sequence cross margin repay sequence, either 'RECENTLY_EXPIRE_FIRST' or 'HIGHEST_RATE_FIRST'
          * @param {string} params.seqStrategy isolated margin repay sequence, either 'RECENTLY_EXPIRE_FIRST' or 'HIGHEST_RATE_FIRST'
+         * @param {string} params.marginMode 'cross' or 'isolated'
```

Comment:
you should write what the default is here, and because this has a default, I think that the type should be `string|undefined`